src/languages/en-US/commands/misc.json

```diff
@@ -374,7 +412,7 @@
 		"examples": [
 			"HE COMES"
 		],
-		"reminder": "H̡̰͉̜̰̠̣͆ͣ̒ͯ̽̃E̪̱̬̪ͣ̓̈́̐͗́̑ͩ͠ͅ ̞͍͍̪̟͚̩̃̀ͯ̓ͥͪͦ̎͜C̣̮̄ͩͤͬͥ͆͛͆͢͞O̲͙̟͎̙͑̿̋̚M̛̖͙͖̣͇̊ͦ̽ͭ͋̑̂̾Ē̸̳͎͖̟̿̆S̱̭̩̳̠̜ͨ̔̿ͅ"
+		"reminder": "H̡̰͉̜̰̠̣͆ͣ̒ͯ̽̃E̪̱̬̪ͣ̓̈́̐͗́̑ͩ͠ͅ ̞͍͍̪̟͚̩̃̀ͯ̓ͥͪͦ̎͜C̣̮̄ͩͤͬͥ͆͛͆͢͞O̲͙̟͎̙͑̿̋̚M̛̖͙͖̣͇̊ͦ̽ͭ͋̑̂̾Ē̸̳͎͖̟̿̆S̱̭̩̳̠̜ͨ̔̿ͅ"
```

Comment:
Why was this changed?

Reply:
Why was this resolved without being answered @Stitch07 ?

Reply:
It wasn't changed, just git merge messing up the formatting.